/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- diffy (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:6:in `<main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- diffy (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:6:in `<main>'
/Users/yknx4/.bash_profile: line 2: brew: command not found
/Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `read': No such file or directory @ rb_sysopen - config.json (Errno::ENOENT)
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `<main>'
/Users/yknx4/.bash_profile: line 2: brew: command not found
/Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `read': No such file or directory @ rb_sysopen - config.json (Errno::ENOENT)
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `<main>'
/Users/yknx4/.bash_profile: line 2: brew: command not found
/Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `read': No such file or directory @ rb_sysopen - config.json (Errno::ENOENT)
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `<main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- diffy (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:6:in `<main>'
/Users/yknx4/Projects/Scripts/check_updates/check.sh: line 2: rbenv: command not found
/Users/yknx4/Projects/Scripts/check_updates/check.sh: line 3: rbenv: command not found
/Users/yknx4/Projects/Scripts/check_updates/check.sh: line 4: rbenv: command not found
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- diffy (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:6:in `<main>'
/Users/yknx4/Projects/Scripts/check_updates/check.sh: line 2: rbenv: command not found
/Users/yknx4/Projects/Scripts/check_updates/check.sh: line 3: rbenv: command not found
/Users/yknx4/Projects/Scripts/check_updates/check.sh: line 4: rbenv: command not found
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- diffy (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:6:in `<main>'
/Users/yknx4/Projects/Scripts/check_updates/check.sh: line 2: rbenv: command not found
/Users/yknx4/Projects/Scripts/check_updates/check.sh: line 3: rbenv: command not found
/Users/yknx4/Projects/Scripts/check_updates/check.sh: line 4: rbenv: command not found
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- diffy (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:6:in `<main>'
/Users/yknx4/Projects/Scripts/check_updates/check.sh: line 2: rbenv: command not found
/Users/yknx4/Projects/Scripts/check_updates/check.sh: line 3: rbenv: command not found
/Users/yknx4/Projects/Scripts/check_updates/check.sh: line 4: rbenv: command not found
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- diffy (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:6:in `<main>'
/Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `read': No such file or directory @ rb_sysopen - config.json (Errno::ENOENT)
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `<main>'
/Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `read': No such file or directory @ rb_sysopen - config.json (Errno::ENOENT)
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `<main>'
/Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `read': No such file or directory @ rb_sysopen - config.json (Errno::ENOENT)
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `<main>'
/Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `read': No such file or directory @ rb_sysopen - config.json (Errno::ENOENT)
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `<main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- diffy (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:6:in `<main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- diffy (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:6:in `<main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- diffy (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:6:in `<main>'
/System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- diffy (LoadError)
	from /System/Library/Frameworks/Ruby.framework/Versions/2.0/usr/lib/ruby/2.0.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:6:in `<main>'
/Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `read': No such file or directory @ rb_sysopen - config.json (Errno::ENOENT)
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `<main>'
/Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `read': No such file or directory @ rb_sysopen - config.json (Errno::ENOENT)
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `<main>'
/Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `read': No such file or directory @ rb_sysopen - config.json (Errno::ENOENT)
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `<main>'
/Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `read': No such file or directory @ rb_sysopen - config.json (Errno::ENOENT)
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:55:in `<main>'
sh: terminal-notifier: command not found
sh: terminal-notifier: command not found
sh: terminal-notifier: command not found
sh: terminal-notifier: command not found
sh: terminal-notifier: command not found
sh: terminal-notifier: command not found
sh: terminal-notifier: command not found
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:880:in `initialize': execution expired (Net::OpenTimeout)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:880:in `open'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:880:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to sccn.vn:80 (Network is unreachable - connect(2) for "sccn.vn" port 80) (Errno::ENETUNREACH)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
/Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:882:in `rescue in block in connect': Failed to open TCP connection to enfree.com:80 (getaddrinfo: nodename nor servname provided, or not known) (SocketError)
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:879:in `block in connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:91:in `block in timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/timeout.rb:101:in `timeout'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:878:in `connect'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:863:in `do_start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:852:in `start'
	from /Users/yknx4/.rbenv/versions/2.3.0/lib/ruby/2.3.0/net/http.rb:584:in `start'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:78:in `block in <main>'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `each'
	from /Users/yknx4/Projects/Scripts/check_updates/check.rb:66:in `<main>'
